src/auth_db/test.rs

```diff
@@ -138,21 +138,11 @@ fn serialize_bounce_subtype() {
 fn get_bounces() {
     let settings = Settings::new().expect("config error");
     let db = DbClient::new(&settings);
-    if let Err(error) = db.get_bounces("foo@example.com") {
+    if let Err(error) = db.get_bounces(&"foo@example.com".parse().unwrap()) {
         assert!(false, format!("{}", error));
     }
 }
 
-#[test]
-fn get_bounces_invalid_address() {
-    let settings = Settings::new().expect("config error");
-    let db = DbClient::new(&settings);
-    match db.get_bounces("") {
-        Ok(_) => assert!(false, "DbClient::get_bounces should have failed"),
-        Err(error) => assert_eq!(format!("{}", error), "400 Bad Request"),
-    }
-}
```

Comment:
Here we can delete the entire test case because it's not possible to construct an invalid `EmailAddress` instance to pass to `db.get_bounces`.